sklearn/preprocessing/tests/test_polynomial.py

```diff
@@ -552,6 +552,37 @@ def test_polynomial_features_csr_X(deg, include_bias, interaction_only, dtype):
     assert_array_almost_equal(Xt_csr.A, Xt_dense)
 
 
+@pytest.mark.parametrize("columns", [1, 2, 3, 1000])
+def test_polynomial_features_csr_wide(columns):
+    x = sparse.csr_matrix(([1], ([0], [columns])))
+
+    est = PolynomialFeatures(3)
+    try:
+        est.fit_transform(x)
+    except ValueError:
+        pytest.fail("possible overflow")
```

Comment:
I'm uncomfortable classifying an overflow as a test failure, rather than an error in executing the test.

Reply:
Changed.